modules/apps/asset/asset-display-page-service/src/main/resources/com/liferay/asset/display/page/internal/upgrade/v1_0_0/dependencies/update.sql

```diff
@@ -0,0 +1,22 @@
+create table AssetDisplayPageEntry (
```

Comment:
Do we really want to create this table manually? Why it is not being created by a Service Builder?

Reply:
For this, I used [UpgradeLayoutPrototype](https://github.com/liferay/liferay-portal/blob/master/modules/apps/layout/layout-page-template-service/src/main/java/com/liferay/layout/page/template/internal/upgrade/v1_1_0/UpgradeLayoutPrototype.java) as an example.

The `AssetDisplayPageEntry` table was automatically created when its bundle was deployed in the upgrade process. However, in this case it creates the schema with the latest version (2.0.0) and none of the upgrade processes will actually run. I found out that the upgrade process has to be referenced ([SME req](https://issues.liferay.com/browse/LPP-32186?focusedCommentId=1621068&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-1621068)) in [UpgradeModules](https://github.com/liferay/liferay-portal/blob/master/portal-impl/src/com/liferay/portal/upgrade/v7_1_x/UpgradeModules.java) to make it work. When this is applied, it sets the schema version to `0.0.1` and deploying the portlet will not automatically create the new table.

So I might be wrong, but my assumption is that the current upgrade for `AssetDisplayPageEntry` has never run.

Although, when we actually move the population upgrade step to `journal-service`, this problem will not really affect us.